pkg/cloud/libvirt/client/client.go

```diff
@@ -194,8 +193,12 @@ func (client *libvirtClient) CreateDomain(input CreateDomainInput) error {
 
 	glog.Info("Create ignition configuration")
 
+	arch, err := getHostArchitecture(client.connection)
+	if err != nil {
+		return fmt.Errorf("Error retrieving host architecture: %s", err)
+	}
 	// Both "s390" and "s390x" are linux kernel architectures for Linux on IBM z Systems, and they are for 31-bit and 64-bit respectively.
-	if runtime.GOARCH == "s390x" || runtime.GOARCH == "s390" {
+	if arch == "s390x" || arch == "s390" {
```

Comment:
I think changes to this file were supposed to be in the first patch as well. You can fix this with `git reset HEAD^` (don't hard reset), `git add PATH_TO_FILE`, `git commit --fix COMMIT_ID_OF_FIRST_COMMIT`, `git commit -a -c SHA1_OF_THIS_COMMIT` (you want to note down the SHA1 before doing any of these) and then an interactive rebase on master.

Reply:
This change should be in the patch: Use "getHostArchitecture" instead of "runtime.GOARCH" to get the arch
Right?
I will add this change to the patch about arch.

Reply:
> This change should be in the patch: Use "getHostArchitecture" instead of "runtime.GOARCH" to get the arch
> Right?

right.

Reply:
done.